test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageExtractorTests.cs

```diff
@@ -27,19 +30,19 @@
 
 namespace NuGet.Packaging.Test
 {
-    using LocalPackageArchiveDownloader = NuGet.Protocol.LocalPackageArchiveDownloader;
+    using LocalPackageArchiveDownloader = Protocol.LocalPackageArchiveDownloader;
 
     [Collection(SigningTestsCollection.Name)]
     public class PackageExtractorTests
     {
-        private static ClientPolicyContext _defaultContext = ClientPolicyContext.GetClientPolicy(NullSettings.Instance, NullLogger.Instance);
```

Comment:
These renames resolve warnings that `static` or `const` fields shouldn't be `_`-prefixed.